src/lib/Preview.js

```diff
@@ -109,9 +109,6 @@ class Preview extends EventEmitter {
     /** @property {Object} - Map of disabled viewer names */
     disabledViewers = {};
 
-    /** @property {string} - Access token */
-    token = '';
-
```

Comment:
Was the token originally a string and you're converting it to a function that has to be passed in?

Reply:
We're currently not using this property at all. And the token is consumed by the viewer, so we actually don't need to store it in Preview instance.